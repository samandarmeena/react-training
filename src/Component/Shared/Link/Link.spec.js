import React from "react";
import { mount } from "enzyme";
import Link from "./Link";
describe(`my component should be run`,() => {
    let wrapper = "";
    beforeAll(() => {
        const props  = {
            anchorText: ""
        }
        wrapper = mount(<Link {...props} />);
    });
    it(`component should be run currectlly`,() => {        
        expect(wrapper).toBeTruthy();
        expect(wrapper).toMatchSnapshot();
    });

});