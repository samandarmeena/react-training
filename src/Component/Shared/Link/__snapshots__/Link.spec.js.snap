// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`my component should be run component should be run currectlly 1`] = `ReactWrapper {}`;
